browser-extension/src/background.js

```diff
@@ -12,13 +12,9 @@ import {
 
 import { frontendHost } from './config.js';
 
-const oversamplingRatioForRecentVideos = 3;
-const oversamplingRatioForOldVideos = 50;
-// Higher means videos recommended can come from further down the recommandation list
-// and returns videos that are more diverse on reload
-
-const recentVideoProportion = 0.75;
-const recentVideoProportionForAdditionalVideos = 0.5;
+const RECENT_VIDEOS_RATIO = 0.75;
+const RECENT_VIDEOS_EXTRA_RATIO = 0.5;
+const BUNDLE_SIZE_F = 3;
```

Comment:
What does `BUNDLE_SIZE_F` stand for?

Reply:
Ha this name caused me trouble : ) It was hard to find a "good" one.

It's a factor used to define the size the returned video bundle: 3 means we want the API to return three times as many videos as we need.

Reply:
BUNDLE_OVERFETCHING_FACTOR ?

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1876/commits/4c1cb676aeca5578c0066e8cac502d78058f761a